pkg/networkservice/common/refresh/client.go

```diff
@@ -35,16 +35,18 @@ import (
 )
 
 type refreshClient struct {
-	connectionTimers map[string]*time.Timer
-	executor         serialize.Executor
+	connectionTimers       map[string]*time.Timer
+	timerContextCancellers map[string]func()
+	executor               serialize.Executor
 }
 
 // NewClient - creates new NetworkServiceClient chain element for refreshing connections before they timeout at the
 // endpoint
 func NewClient() networkservice.NetworkServiceClient {
 	rv := &refreshClient{
-		connectionTimers: make(map[string]*time.Timer),
-		executor:         serialize.Executor{},
+		connectionTimers:       make(map[string]*time.Timer),
+		timerContextCancellers: make(map[string]func()),
+		executor:               serialize.Executor{},
```

Comment:
We do not need to assign value for field `executor`.

Reply:
I'll remove it, thanks.